submissions/task_html-css-popup/style.css

```
@@ -0,0 +1,388 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: Verdana, Geneva, Tahoma, sans-serif;
+}
+
+/* DECLARE .HEADER */
+
+.header {
+    display: flex;
+    justify-content: flex-end;
+    width: calc(100%-80px);
+    height: 50px;
+    margin: 10px 10px;
+    padding: 2px 2px;
+    border: solid black 1px;
+    box-shadow: 1px 2px;
+    background-color:blue;
+}
+
+/* HEADER - Hide dots in the <ul> element*/
+
+.menu__list {
+    list-style: none;
+}
+
+/* HEADER - Style for <a> element */
+
+a {
+    text-decoration-style: dashed;
+    text-decoration-color: white;
+    text-underline-offset: 3px;
+}
+
+a:focus {
+    outline: 3px dashed gold;
+    text-decoration: underline;
+}
+
+a:hover,
+a:active {
+    text-decoration: underline;
+    text-decoration-color: white;
+}
+
+/* DECLARE .MAIN */
+
+.main {
+    display: flex;
+    width: calc(100%-80px);

```

I don't know why you need it, but use spaces and calc will work properly
```suggestion
    width: calc(100% - 80px);

```

---

I've changed width of elements, no more calc.